index.html

```diff
@@ -49,5 +54,204 @@ <h4>I’am a Software Developer</h4>
+        <ul class="works-categories">
+          <li>css</li>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="button">
+          See Projects         
+        </button>
+      </div>
+   </section>
+
+   <section class="Projects"> 
+    <div class="first-project"> 
+        <h1>Profesional Art  <br> Printing Data</h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <a href="#">
+          See Project
+        </a>
+      </div>
+
+      <div class="first-project"> 
+        <h1>Profesional Art  <br> Printing Data</h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <a href="#">
+          See Project
+        </a>
+      </div>
+
+      <div class="first-project"> 
+        <h1>Profesional Art  <br> Printing Data</h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <a href="#">
+          See Project
+        </a>
+      </div>
+
+      <div class="first-project"> 
+        <h1>Profesional Art  <br> Printing Data</h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <a href="#">
+          See Project
+        </a>
+      </div>
+
+      <div class="first-project"> 
+        <h1>Profesional Art 
+           <br> Printing Data
+        </h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <a href="#">
+          See Project
+        </a>
+      </div>
+
+      <div class="first-project"> 
+        <h1>Profesional Art  
+          <br> Printing Data
+        </h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <a href="#">
+          See Project
+        </a>
+      </div>
+   </section>
+
+  <section class="about-me">
+   <div class="about">
+    <h1>About me</h1>
```

Comment:
@Kossi-stack i used that title for my pull request because i already have a pull request with that name when i created the header and headline section, so i felt since i was making a different change and with a different pull request i should use a different title which will be descriptive of the changes i had just made that was why i used that title, however can you explain to me why i should have used the same pull request title that i had used for a previous pull request in the same repository.

Also thanks, i will start my commit messages with capital letter henceforth